frontend/src/ImageProvider.tsx

```diff
@@ -2,22 +2,28 @@ import React from "react";
 
 export class ImageProvider {
 
-    
-
     /** @returns a social media icon for a given platform identifier. */
     static getSocialmediaIcon(platformId: number) {
         if (platformId === 1) {
             return ImageProvider.getImage("facebook-logo");
```

Comment:
Why does it sometimes have the suffix "-logo" and sometimes it doesn't? If we make this consistent, maybe we could use that dynamically somewhere else.

Reply:
Done.